fs/expose/dokan/__init__.py

```diff
@@ -52,19 +58,22 @@
 
 """
 #  Copyright (c) 2009-2010, Cloud Matrix Pty. Ltd.
+#  Copyright (c) 2016-2016, Adrien J. <liryna.stark@gmail.com>.
 #  All rights reserved; available under the terms of the MIT License.
 
 from __future__ import with_statement
 
+import six
 import sys
-
 import os
-import signal
 import errno
 import time
 import stat as statinfo
 import subprocess
-import cPickle
+try:
+    import cPickle
+except ImportError:
+    import pickle as cPickle
```

Comment:
I wonder if this (and also line 1072) should be 'inverted' to:
```python
try:
    import cPickle as pickle
except ImportError:
    pass
```
?
(and then the other references to `cPickle` obviously changed to `pickle` as necessary)